Lib/test/test_faulthandler.py

```diff
@@ -817,6 +817,15 @@ def test_disable_windows_exc_handler(self):
         self.assertEqual(output, [])
         self.assertEqual(exitcode, 0xC0000005)
 
+    def test_cancel_later_no_dump(self):
```

Comment:
Please add a comment like:

```
# bpo-37933: Calling cancel_dump_traceback_later()
# without dump_traceback_later() must not crash
```

Maybe rename the test to test_cancel_without_dump_traceback_later().